Examples/Transform/index.js

```diff
@@ -0,0 +1,167 @@
+
+// Display views and an image with various transforms applied
```

Comment:
@ndpmurphy, please, use the same naming for example js files as in another examples, `TransformApp.js` in this case.
Also example should be added to `Examples/CMakeLists.txt`

Reply:
I have made this change, but can you tell me what is the standard way of running the Examples. I have been manually editing the main.qml.in file to set the source file and moduleName. This works, but I am sure the CMakefiles must provide a better way to launch an example from with QtCreator or from the commandline.

Reply:
@ndpmurphy, here is how you can setup "Run" section in QtCreator to launch examples using `qmlscene`:

<img width="825" alt="screen shot 2018-03-20 at 8 59 39 am" src="https://user-images.githubusercontent.com/5786310/37640175-003a26d8-2c47-11e8-8663-74dc05b26a7b.png">

Reply:
But there is no Image.qml file in the Image directory. Was this supposed to be generated by CMakefile ?

Reply:
@ndpmurphy, this command invoked in `%{buildDir}`, which is build directory. You will get Image.qml there after build.

In `Examples/Image/CMakeLists.txt` you can find following code:
```
configure_file(
  ../Example.qml.in
  ${CMAKE_CURRENT_BINARY_DIR}/${EXAMPLE_NAME}.qml
)
```

It is responsible for generating `Image.qml`

Reply:
That does not work for me. If I open the CMakefiel at the top level, then I can not  get it to build. I have been opening the CMakefile in the ReactQT directory and that works fine for most things, but it does not generate the qml files in Examples. When opening the project, is the CMAkefile at the top levelthe right file to open or is there another project file ?

Reply:
Oh - ignore that I see my mistake now

Reply:
@ndpmurphy, were you able to run the example apps the way I mentioned above?
I usually open root CMakeLists.txt in QtCreator. Then you will be able to build both, ReactQt and Examples projects.